src/cli/cli_tcp.cpp

```diff
@@ -85,6 +85,31 @@ void TcpExample::MbedTlsDebugOutput(void *ctx, int level, const char *file, int
 }
 #endif // OPENTHREAD_CONFIG_TLS_ENABLE
 
+/**
+ * @cli tcp init
+ * @code
+ * tcp init tls
+ * Done
+ * @endcode
+ * @cparam tcp init [@ca{mode}] [@ca{size}]
+ * * The `mode` has three possible values:
+ *   + `tls`: Specifies that the TCP connection between two nodes should also
+ *    use the TLS protocol on top of TCP. When two nodes communicate over TCP,
```

Comment:
Subsequent lines should align with the text above it.  So the ` in this case.  It's currently aligned on the space

Please update throughout